lib/religion/religion.ts

```diff
@@ -2470,7 +2497,8 @@ export const religion: ReligionData = {
             }
           ],
           followers: {
-            description: undefined,
+            description: 'Asclepius is the son of Apollo whose skill in medicine was so great he could ressurect the dead when he was mortal. Zeus struck him down but Asclepius was placed among the stars and became the Physician of the Gods',
```

Comment:
This is meant to be a description of the followers. i.e. "Followers of Asclepius are those with an interest in medicine and the resurrection of the dead."